test/expected/empty.js

```diff
@@ -0,0 +1 @@
+//# sourceMappingURL=empty.map
```

Comment:
Are these files used?  Looks like we are using `test/fixtures/empty/empty.js` in `test/test.js`, but I don't see where `test/expected/empty.js`, `test/expected/empty-no-sourcemap.js`, `test/expected/empty.map` are used...